src/utils/legacy-settings.js

```diff
@@ -61,6 +61,7 @@ const LegacySettingIds = {
   DELETED_MESSAGES: 'deletedMessages',
   BLACKLIST_KEYWORDS: 'blacklistKeywords',
   HIGHLIGHT_KEYWORDS: 'highlightKeywords',
+  HIDE_HYPECHAT: 'hideHypeChat',
```

Comment:
Undo any changes made to this file, this is used to migrate the old settings into a dictionary.